FSharpLu.Json/WithFunctor.fs

```diff
@@ -26,6 +26,16 @@ type With< ^S when ^S : (static member settings : JsonSerializerSettings)
         let json = JsonConvert.SerializeObject(obj, formatting, settings)
         System.IO.File.WriteAllText(file, json)
 
+    /// Serialize an object to Json with the specified converter and write the result to a stream
+    static member inline public serializeToStream (stream:System.IO.Stream) (obj:^T) =
+        let settings = (^S:(static member settings : JsonSerializerSettings)())
+        let formatting = (^S:(static member formatting : Formatting)())
+        let serializer = JsonSerializer.Create(settings)
+        serializer.Formatting <- formatting
+        use streamWriter = new System.IO.StreamWriter(stream)
```

Comment:
This is will close the stream when the function returns.
This should probably leave the stream open instead

Reply:
I don't know a lot about the Stream/StreamWriter/TextWriter APIs in .NET unfortunately. How would I leave the stream open? Should the function take another type than Stream instead?

Reply:
There is a constructor of StreamWriter that allows you to specify if you want to[ leave the stream open](https://docs.microsoft.com/en-us/dotnet/api/system.io.streamwriter.-ctor?view=netframework-4.8#System_IO_StreamWriter__ctor_System_IO_Stream_System_Text_Encoding_System_Int32_System_Boolean_) 
```F#
use streamWriter = new System.IO.StreamWriter(stream, System.Text.UTF8Encoding.UTF8, 1024, true )
```

Reply:
Thanks! Just updated with your suggestion

Reply:
One last can you put 1024 in a constant